drivers/video/video_mcux_mipi_csi2rx.c

```diff
@@ -7,19 +7,18 @@
 #define DT_DRV_COMPAT nxp_mipi_csi2rx
 
 #include <zephyr/drivers/video.h>
+#include <zephyr/drivers/video-controls.h>
 #include <zephyr/kernel.h>
 #include <zephyr/logging/log.h>
+#include <soc.h>
 
 #include <fsl_mipi_csi2rx.h>
 
 LOG_MODULE_REGISTER(video_mipi_csi2rx, CONFIG_VIDEO_LOG_LEVEL);
 
-/*
- * Two data lanes are set by default as 2-lanes camera sensors are
- * more common and more performant but single lane is also supported.
- */
-#define DEFAULT_MIPI_CSI_NUM_LANES 2
-#define DEFAULT_CAMERA_FRAME_RATE  30
+#define MAX_SUPPORTED_PIXEL_RATE MHZ(96)
+
+#define ABS(a, b) (a > b ? a - b : b - a)
```

Comment:
The `ABS()` macro is sometimes defined as `a > 0 ? a : -a` (like C's [`abs()`](https://www.man7.org/linux/man-pages/man3/abs.3.html) from `<stdlib.h>`) 
I proposed an `ABS_DIFF(a, b)` in `util.h` [there](https://github.com/zephyrproject-rtos/zephyr/pull/79482/files#diff-c5b30d0a3cf86108357ba5a1c2a12f48f67d2f375e2adbe7549296f3c720f52d).

I do not think there is an issue with having it defined in `video_mcux_mipi_csi2rx.c` as it is not an `#include` file so no side effect.